cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -0,0 +1,137 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.rest.gen_data import users as _users
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.rest import assert_response
+from cfme.utils.wait import wait_for_decorator
+
+pytestmark = [test_requirements.report, pytest.mark.tier(3), pytest.mark.sauce]
+
+
+@pytest.fixture(scope="function")
+def rbac_api(appliance, request):
+    user, user_data = _users(
+        request, appliance, password="smartvm", group="EvmGroup-user"
+    )
+    yield appliance.new_rest_api_instance(
+        entry_point=appliance.rest_api._entry_point,
+        auth=(user[0].userid, user_data[0]["password"]),
+    )
+    appliance.rest_api.collections.users.action.delete(id=user[0].id)
+
```

Comment:
What do you think instead of going with existing implementation we can update it as below? It would be more readable.
```
new_rest_instance = appliance.new_rest_api_instance(
        entry_point=appliance.rest_api._entry_point,
        auth=(user[0].userid, user_data[0]["password"]),
    )
yield new_rest_instance
    appliance.rest_api.collections.users.action.delete() # (Add teardown here)
```

Reply:
There's not going to be any use of new_rest_api_instance any further than this, I don't think it's required. And this is creating a rest_api instance since I need to access API with a user other than admin, there's no need for teardown with rest_api instance afaik.